packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -200,6 +200,19 @@ class _ProductPageState extends State<NewProductPage> {
   }
 
   Widget _buildProductBody(BuildContext context) {
+    final List<Widget> listItems = <Widget>[];
+    listItems.add(
+      Align(
+        heightFactor: 0.7,
+        alignment: Alignment.topLeft,
+        child: _buildProductImagesCarousel(context),
+      ),
+    );
+    listItems.add(_buildSummaryCard());
+    return ListView(children: listItems);
+  }
```

Comment:
Just a matter of preference, but what about directly returning the ListView

``` dart
  return ListView(
    children: <Widget>[
      Align(),
      _buildSummaryCard(),
    ],
  ); 
```

Reply:
yeah I also thought about that, I think your suggestion is fine to have now but the list items might increase in the future which might make inlining hard to read, which is why I didn't inline already. That said, I incorporated your changes, can change it back if needed later.